<?xml version="1.0"?>

<launch>
  <param name="video_output"  value="true" />   <!-- true == Publish video out, false == only publish co-ordinates-->
  <param name="face_classifier"  value="$(find marty_face_tracker)/classifiers/haarcascade_face.xml" />
  <param name="eye_classifier"  value="$(find marty_face_tracker)/classifiers/haarcascade_eye.xml" />
  <param name="smile_classifier"  value="$(find marty_face_tracker)/classifiers/haarcascade_smile.xml" />
  <rosparam file="$(find marty_face_tracker)/cfg/detection_params.yaml" command="load" />

  <node pkg="marty_face_tracker" type="marty_face_tracker" name="face_tracking"
  output="screen"/>

</launch>
